client/www/scripts/modules/common/common.directives.js

```diff
@@ -290,64 +265,42 @@ Common.directive('slCommonAppControllerMenu', [
     }
   }
 ]);
+
 /*
 *
 *   Common Instance Tabs View
 *
 * */
 Common.directive('slCommonInstanceTabsView', [
-  'IAService',
-  function(IAService) {
+  '$http',
```

Comment:
These requirements can be removed, since they aren't being used anywhere in the directive.